### PR TITLE
Skip docker tests when not Ansible 2.x

### PR DESCRIPTION
**Pull request by retr0h**
Fixes: #299

### CODE REVIEW

**Comment by retr0h** [comment]
```
  py27-ansible19-unit: commands succeeded
  py27-ansible19-functional: commands succeeded
  py27-ansible20-unit: commands succeeded
  py27-ansible20-functional: commands succeeded
ERROR:   py27-ansible21-unit: commands failed
ERROR:   py27-ansible21-functional: commands failed
  syntax: commands succeeded
  docs: commands succeeded
```

File: tests/unit/provisioner/test_vagrantprovisioner.py [hunk 1]
```diff
@@ -62,8 +67,7 @@ def molecule_file(tmpdir, request):
 
     pbook = d.join(os.extsep.join(('playbook', 'yml')))
     data = [{'hosts': 'all', 'tasks': [{'command': 'echo'}]}]
-
-    pbook.write(data)
+    pbook.write(yaml.dump(data))
```
Comment (abrown-sg) [inline tests/unit/provisioner/test_vagrantprovisioner.py]: Possibly want to use safe_dump here? Not sure how Ansible will deal with the extra info that can be output with just dump.

Comment (retr0h) [inline tests/unit/provisioner/test_vagrantprovisioner.py]: What extra info that can be output?  It's just a simple dict.

Comment (abrown-sg) [inline tests/unit/provisioner/test_vagrantprovisioner.py]: Ah, right, I think it's the anyconfig stuff that introduced the extra info into the molecule dicts when doing a dump.

**Comment by abrown-sg** [comment]
👍

**Pull request merged**